packages/wp-now/src/pool.ts

```diff
@@ -0,0 +1,295 @@
+type instance = any;
```

Comment:
I immediately have a question in my head: "an `instance` of what?". Let's include the answer in the type name.

Reply:
Thanks! By the way, why `any`? `@php-wasm/node` and `@php-wasm/universal` ship types like `UniversalPHP` or `NodePHP` – would either be applicable here?